pyproject.toml

```diff
@@ -27,7 +27,7 @@ copier = "copier.__main__:copier_app_run"
 "Bug Tracker" = "https://github.com/copier-org/copier/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0" # HACK https://github.com/PyCQA/isort/issues/1945
+python = ">=3.8,<4.0"                                               # HACK https://github.com/PyCQA/isort/issues/1945
```

Comment:
Wow, what's this extra space :joy:? Does it try to align comment on the far right or something?

Reply:
Wow amazing, I didn't see this. Let me check

Reply:
Fun, it turns out that the issue itself was closed long ago, so I removed the hack.